impeller/renderer/backend/gles/texture_gles.cc

```diff
@@ -154,7 +154,9 @@ struct TexImage2DData {
 };
 
 // |Texture|
-bool TextureGLES::SetContents(const uint8_t* contents, size_t length) {
+bool TextureGLES::SetContents(const uint8_t* contents,
```

Comment:
Verify that the slice is 0 if the descriptor format is anything but kTextureCube?

Reply:
Done.